scripts/build.js

```diff
@@ -141,18 +141,3 @@ nextTask("Building source", async () => {
     },
   });
 });
-
-nextTask("Cleanup", async () => {
-  fs.rmSync(
-    path.join(
-      __dirname,
-      "../packages",
-      commanderOpts.prefix.slice(0, -1),
-      "/src/react"
-    ),
-    {
-      force: true,
-      recursive: true,
-    }
-  );
-});
```

Comment:
Hey @SalihuDickson, is there a reason why not to remove the `src/react` after the build is completed? I actually do not want any generated files to be present in the source code (even if they are being ignored). This has to do with how different tools interact with the generated files & ignore them some may use .gitignore some may not & we will have to configure it manually. Moreover, I do not see any pros to keeping them if any we can keep them.

Reply:
well removing the react dir makes general debugging a bit harder, considering that you won't have the source for the react components the only errors we can catch are compile time errors. After building the components I think it generally makes sense to be able to validate the source, especially considering that generating the source like this is pretty fragile and even minor, unintuitive changes can cause issues. Maybe we can automate these tests and make manually checking the source unnecessary but for now, considering there aren't any real costs to keeping the source dir, I think we should have to justify deleting it.

Reply:
@anuragxxd I too do agree with @SalihuDickson at this specially because it's not cause build issues because of it as it was doing previously. Will need to be sure but yeah unless it doesn't cause any problems while dev there is no harm in keeping it but that being said I don't think we will be visiting build scripts too often hence most wouldn't need the debugging. How about adding another flag in the build command to generate reach components and we don't add that flag in package.json. This way if someone has --react-debug on when building they can look at source code.

How does that sound?

Reply:
Introducing the new flag is an overhead for now (but we have something like this). Right now if it sounds good to you @SalihuDickson we can remove it maybe from the src. I am not sure why but the idea of having generated code inside the src doesn't seem very right to me as it is not considered the best practice community wide. Though I understand it is not being committed!